nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/AbstractExecuteSQL.java

```diff
@@ -82,6 +84,16 @@
             .identifiesControllerService(DBCPService.class)
             .build();
 
+    public static final PropertyDescriptor SQL_PRE_QUERY = new PropertyDescriptor.Builder()
+            .name("sql-pre-query")
+            .displayName("SQL Pre-Query")
+            .description("SQL pre-query to execute. Semicolon-delimited list of queries. "
```

Comment:
"SQL pre-query to execute" is duplicative with the name. Can you explain in another way its intent? Perhaps something like, "Queries that will be executed before each intended query is executed."

Reply:
@colindean I understand why it was done this way. This functionality is being ported from the HIVE processor where the description begins: `HiveQL pre-query to execute...`. The same for the `Note` comment below.
That doesn't mean it couldn't be clarified, but it is currently internally consistent.

@yjhyjhyjh0 How about, "A semicolon-delimited list of queries executed executed before the main SQL query is executed. Results/outputs from these queries will be suppressed if there are no errors."

Reply:
Sounds good. Can you squash your next commit?

Reply:
Sounds great to me.
I'll also modify Hive description and squash the commit.